scripts/check_frontend_coverage.py

```diff
@@ -537,35 +590,38 @@ def check_coverage_changes():
                         ' file does not exist.'.format(LCOV_FILE_PATH))
 
     stanzas = get_stanzas_from_lcov_file()
-    whitelist = list(FULLY_COVERED_FILENAMES)
+    blacklist = list(NOT_FULLY_COVERED_FILENAMES)
```

Comment:
Can you please update the function docstring accordingly?

Reply:
Done.